.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,48 @@
+name: Push to ECR
+
+on: [push, pull_request]
+
+jobs:
+   deploy:
+    runs-on: ubuntu-latest  
+    steps:
+    
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Build the Docker-compose file
+      run: docker-compose -f "docker-compose.yaml" up -d --build
+
+    - name: Show containers
+      run: docker ps -a
+    
+    - name: Configure AWS credentials
+      if: ${{ github.ref == 'refs/heads/master' }}
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Login to Amazon ECR
+      if: ${{ github.ref == 'refs/heads/master' }}
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+      with:
+          registry-type: public
+    
+    - name: Set outputs
+      id: vars
```

Comment:
Put some prefixes that will help in future to distinguish images like gs- or rc- etc same like above